src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2423,6 +2423,19 @@
     public static final int NOT_EQUAL = GeneratedJavaTokenTypes.NOT_EQUAL;
     /**
      * The {@code ==} (equal) operator.
+     * <p>For example:</p>
+     * <pre>
+     * return a == b;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--LITERAL_RETURN -&gt; return
```

Comment:
remove first `|` it is not required and it leads to nowhere.

Reply:
I was also thinking the same, but wanted a review on the same. Thank you I will update the PR.